src/modules/paymentwall/paymentwall.php

```diff
@@ -23,7 +24,7 @@ public function __construct()
         $this->confirmUninstall = $this->l('Are you sure you want to uninstall, and remove all information regarding this module?');
 
         $this->initPaymentwallConfig();
-        $this->context->controller->addCSS($this->_path . 'css/paymentwall.css');
```

Comment:
Please update Paymentwall logos.
You can take them from [here](https://www.paymentwall.com/en/assets).